### PR TITLE
Add useful setters to ResampleOptions struct

### PR DESCRIPTION
**Pull request by askoufis**
These are fields I need to change for a project I'm working on, and they seem the most useful out of all the resample options anyway.

Changing the sample rate is not currently supported by ffms2
https://github.com/FFMS/ffms2/blob/master/src/core/audiosource.cpp#L201

### CODE REVIEW

File: src/resample.rs [hunk 1]
```diff
@@ -128,6 +128,24 @@ impl ResampleOptions {
     pub(crate) fn as_ptr(&self) -> *const FFMS_ResampleOptions {
         &self.resample
     }
+
+    pub fn set_channel_layout(&mut self, channel_layout: i64) {
+        let mut resample = self.resample;
+        resample.ChannelLayout = channel_layout;
```
Comment (Luni-4) [inline src/resample.rs]: We can reduce to a single line: `self.resample.ChannelLayout = channel.layout;`

File: src/resample.rs [hunk 2]
```diff
@@ -128,6 +128,24 @@ impl ResampleOptions {
...
+        self.resample = resample;
+    }
+
+    pub fn set_sample_format(&mut self, sample_format: &SampleFormat) {
+        let mut resample = self.resample;
+        resample.SampleFormat = SampleFormat::to_sample_format(sample_format);
```
Comment (Luni-4) [inline src/resample.rs]: Same as above: `self.resample.SampleFormat = SampleFormat::to_sample_format(sample_format);`

**Review by Luni-4 — CHANGES_REQUESTED** [review]
Thank you again! :)

Just a few nits

**Comment by askoufis** [comment]
Done

**Review by Luni-4 — APPROVED** [review]
Thanks a lot! :)

**Pull request merged**